pulsar-broker/src/test/java/org/apache/pulsar/functions/worker/PulsarFunctionE2ESecurityTest.java

```diff
@@ -177,10 +177,10 @@ void setup(Method method) throws Exception {
 
         ClientBuilder clientBuilder = PulsarClient.builder().serviceUrl(this.workerConfig.getPulsarServiceUrl())
                 .operationTimeout(1000, TimeUnit.MILLISECONDS);
-        if (isNotBlank(workerConfig.getClientAuthenticationPlugin())
-                && isNotBlank(workerConfig.getClientAuthenticationParameters())) {
-            clientBuilder.authentication(workerConfig.getClientAuthenticationPlugin(),
-                    workerConfig.getClientAuthenticationParameters());
+        if (isNotBlank(workerConfig.getBrokerClientAuthenticationPlugin())
```

Comment:
You need to consider backward compatibility. You can add `getBrokerClientAuthenticationPlugin()` method in WorkerConfig.

```
String getBrokerClientAuthenticationPlugin() {
    if (null == brokerClientAuthenticationParameters) {
        return clientAuthenticationParameters;
    } else {
        return brokerClientAuthenticationParameters;
    }
}
```
We should do the same thing for `getBrokerClientAuthenticationPlugin`.